lib/vmpooler/api/v1.rb

```diff
@@ -28,6 +28,10 @@ def pools
       Vmpooler::API.settings.config[:pools]
     end
 
+    def pools_at_startup
```

Comment:
Nitpick, but is this meant to be pools loaded from config file, or all pools available at startup?